plex-argo/README.md

```diff
@@ -1,40 +1,55 @@
-# Create a free CloudFlare Argo tunnel to your Plex Media Server
-This enables fast remote access to your Plex Media Server, even behind a Carrier Grade NAT or VPN with no port-forwarding ability.
+# Create a **free** Cloudflare Argo Tunnel to your Plex Media Server
+
+Enables fast remote access to your Plex Media Server, even behind a carrier grade NAT or VPN with no port-forwarding ability.
 
 ## CloudFlare Argo
-Read the [CloudFlare blog post on Free Argo tunnels](https://blog.cloudflare.com/a-free-argo-tunnel-for-your-next-project/).
-
-TL;DR: Free **TryCloudFlare** Argo Tunnels:
- - Have a unique URL per session (i.e. apple-bali-matters-local.trycloudflare.com)
- - Support both HTTP:80 and HTTPS:443
- - Are free to use and have no bandwidth charges associated with them
- - Require no account or authentication
- - Operate much like a Reverse SSH tunnel + nginx on a remote VPS, except you dont need to do any setup.
- 
+
+Read the [The Cloudflare Blog - A free Argo Tunnel for your next project](https://blog.cloudflare.com/a-free-argo-tunnel-for-your-next-project/).
+
+TL;DR - Free **TryCloudFlare** Argo Tunnel features:
+ - Operate much like a Reverse SSH tunnel + nginx on a remote VPS
+ - Unique URLs per session (i.e. apple-bali-matters-local.trycloudflare.com)
+ - Support for http:80 & https:443
+ - Free to use and no bandwidth restrictions
+ - No account or authentication requirements
+ - Simplier setup with _much_ less overhead
  
 ## Plex Custom Connection URLs
-When you specity a Custom Connection URL in your Plex Media Server, it will publish that URL in the PlexAPI, allowing all your clients to discover alternative paths to your server. This Published information can be seen for your server by going to `https://plex.tv/api/resources?X-Plex-Token=YOUR_API_TOKEN`
-This API token is refreshed every time your Plex Media Server restarts.
 
-You can obtain your current API Token and see your current customConnections URL in the Plex API with the following powershell snippet:
-    
+When you specify a custom connection URL in your Plex Media Server, it will publish that URL in the PlexAPI. This allows all your clients to discover alternative paths to your server. 
+
+This Published information can be seen for your server by going to `https://plex.tv/api/resources?X-Plex-Token=YOUR_API_TOKEN`
+
+**NOTE:** This API token refreshes each time your Plex Media Server restarts.
```

Comment:
refreshes-->changes (since the token itself changes, what it references refreshes)